docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -335,7 +335,9 @@ metrics-category=["BLOCKCHAIN","PEERS","PROCESS"]
 ```
 
 Comma separated list of categories for which to track metrics. The default is all categories: 
-`BIG_QUEUE`, `BLOCKCHAIN`, `EXECUTORS`, `JVM`, `NETWORK`, `PEERS`, `PROCESS`, `ROCKSDB`, `RPC`, `SYNCHRONIZER`. 
+`BIG_QUEUE`, `BLOCKCHAIN`, `EXECUTORS`, `JVM`, `NETWORK`, `PEERS`, `PROCESS`, `RPC`, `SYNCHRONIZER`. 
+
+Additional options are `KVSTORE_ROCKSDB` and `KVSTORE_ROCKSDB_STATS`. When you use these options, the `--key-value-storage` option must also be specified.
```

Comment:
@mark-terry can you confirm that you have to also specify the keyvaluestorage option

Reply:
It's not mandatory.  These metrics are available for the default rocksdb key-value store.

Reply:
Mark and I agreed the issue I was having was most likely an outlier. I removed the part about `key-value-storage` being required.